docs/news/index.qmd

```diff
@@ -2,6 +2,36 @@
 
 This page provides the release notes associated with each release of RStudio and Posit Workbench. Please contact customer support (<a href="mailto:support@posit.co">support@posit.co</a>) for questions about the described changes.
 
+## RStudio 2024.04.1
+
+**"Chocolate Cosmos"**
+
+>Date: 2024-05-11
```

Comment:
One question, is this date intended to match the binaries date or the date of the release?

Reply:
Looks like it should be the release date so I updated it to be today.